ui/src/Roles/RoleClient.ts

```diff
@@ -16,10 +16,13 @@
  */
 
 import Axios, {AxiosResponse} from 'axios';
-import {RoleAddRequest} from './RoleAddRequest';
-import {RoleEditRequest} from './RoleEditRequest';
 import {TraitClient} from '../Traits/TraitClient';
 import {getToken} from '../Auth/TokenProvider';
+import MatomoEvents from "../Matomo/MatomoEvents";
+import {Trait} from "../Traits/Trait";
+import {TraitAddRequest} from "../Traits/TraitAddRequest";
+import {Space} from "../Space/Space";
+import {TraitEditRequest} from "../Traits/TraitEditRequest";
```

Comment:
Please fix lint errors (double quotes)

Reply:
Also, the traits folder and files shouldn't exist anymore... why is there traits?